src/cli/common.rs

```diff
@@ -56,12 +56,18 @@ pub(crate) enum Confirm {
     Advanced,
 }
 
-pub(crate) fn confirm_advanced() -> Result<Confirm> {
+pub(crate) fn confirm_advanced(customised_install: &bool) -> Result<Confirm> {
     writeln!(process().stdout().lock())?;
-    writeln!(
-        process().stdout().lock(),
-        "1) Proceed with installation (default - just press enter)"
-    )?;
+    let one_prompt = match customised_install {
+        true => "Proceed with custom install. Press Enter to run custom install",
+        false => "Proceed with standard install. Press Enter to run (1)",
+    };
+    writeln!(process().stdout().lock(), "{one_prompt}")?;
+    let first_option = match customised_install {
+        true => "1) Proceed with customised install",
```

Comment:
I think we should keep the `(default - just press enter)` and just change the wording before that in the first option. I'd suggest "Proceed with selected options (default - just press enter)" in the customized case.